include/jemalloc/internal/tsd.h

```diff
@@ -84,25 +93,32 @@ typedef void (*test_callback_t)(int *);
     O(witness_tsd,              witness_tsd_t,		witness_tsdn_t)	\
     MALLOC_TEST_TSD
 
+/* THREAD_EVENT_MIN_WAIT should not exceed the minimal allocation usize. */
+#define THREAD_EVENT_MIN_WAIT ((uint64_t)1U)
```

Comment:
How would you like to update the comment? I updated `THREAD_EVENT_MIN_WAIT` => `THREAD_EVENT_MIN_START_WAIT`. It should still not exceed the minimal allocation usize, since whenever we set wait times to be `THREAD_EVENT_MIN_START_WAIT`, our goal is to delay the event to the next allocation.

Reply:
I see. Makes sense. I was thinking about something different. Let's keep as is.